src/pudl/logging_helpers.py

```diff
@@ -3,33 +3,39 @@
 import logging
 
 import coloredlogs
+from dagster import get_dagster_logger
 
 
 def get_logger(name: str):
     """Helper function to append 'catalystcoop' to logger name and return logger."""
-    return logging.getLogger(f"catalystcoop.{name}")
+    return get_dagster_logger(f"catalystcoop.{name}")
 
 
 def configure_root_logger(
     logfile: str | None = None,
     loglevel: str = "INFO",
-    dependency_loglevels: dict[str, int] = {"numba": logging.WARNING},
-):
+    dependency_loglevels: dict[str, int] | None = None,
+    propagate: bool = False,
```

Comment:
Add `propagate` as a new parameter so we can use it in the pytest configuration.